components/template.tsx

```diff
@@ -212,15 +214,15 @@ export default ({ title = 'NewsKit Demo', children }) => (
           font-style: italic;
           font-weight: 700;
         }
-        
+
         body {
           margin: 0;
           padding: 0;
         }
       `}
     />
-    <ThemeProvider theme={theme}>
+    <NewsKitProvider theme={theme}>
```

Comment:
I am not sure if you need this one, in the whole app you must ave only 1 newskit provider and from my understanding, there is already one coming from `useThemeSwitcher`.

Reply:
Updated